system/greenhouse-ccloud/templates/opentelemetry-pluginpreset.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if .Values.openTelemetry.enabled -}}
+{{- range $r := $.Values.openTelemetry.clusters }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: otel-basic-auth-{{ $r.region }}
+  namespace: {{ .Release.Namespace }}
+data:
+  password: {{ required ".Values.openTelemetry.openSearchLogs.password missing" $r.openSearchLogs.password | b64enc }}
```

Comment:
The required text is misleading maybe ".Values.openTelemetry.clusters[].openSearchLogs.password missing"